client/components/domains/use-my-domain/index.jsx

```diff
@@ -133,9 +133,9 @@ function UseMyDomain( props ) {
 		setIsFetchingAvailability( true );
 		let inboundTransferStatusResult = {};
 		try {
-			inboundTransferStatusResult = await wpcom
-				.undocumented()
-				.getInboundTransferStatus( domainName );
+			inboundTransferStatusResult = await wpcom.req.get(
+				`/domains/${ encodeURIComponent( domainName ) }/inbound-transfer-status`
+			);
 		} catch {}
```

Comment:
There's a bit more concise way to write this:
```js
const inboundTransferStatusResult = await wpcom.req.get( ... ).catch( () => ( {} ) );
```

Reply:
Ah you're right, thanks. That is a lot better! Added in 53435e8